CHANGELOG.md

```diff
@@ -28,6 +28,13 @@
 
   ([PR #N](https://github.com/alphagov/govuk-frontend/pull/N))
 
+- Allow attributes on checkboxes/radios
```

Comment:
Since this is adding a new feature this changelog entry should be under the feature title.


- https://github.com/alphagov/govuk-frontend/blob/master/CONTRIBUTING.md#updating-changelog

Reply:
@nickcolley No problem, moved it to "New features"